appservice/src/createAppService/AppInsightsCreateStep.ts

```diff
@@ -17,36 +18,39 @@ import { IAppServiceWizardContext } from './IAppServiceWizardContext';
 export class AppInsightsCreateStep extends AzureWizardExecuteStepWithActivityOutput<IAppServiceWizardContext & Partial<ExecuteActivityContext>> {
     public priority: number = 135;
     public stepName: string = 'AppInsightsCreateStep';
-    private _usedExistingAppInsights: boolean = false;
     private _skippedCreate: boolean = false;
 
-    protected getTreeItemLabel(context: IAppServiceWizardContext): string {
-        const aiName: string = nonNullProp(context, 'newAppInsightsName');
-        let message: string = l10n.t('Create application insights "{0}"', aiName);
+    protected getOutputLogSuccess(context: IAppServiceWizardContext): string {
+        let message: string = l10n.t('Successfully created application insights "{0}".', nonNullProp(context, 'newAppInsightsName'));
         if (this._skippedCreate) {
-            message = l10n.t('Skipping application insights creation.');
-        } else if (this._usedExistingAppInsights) {
-            message = l10n.t('Using existing application insights "{0}"', aiName);
+            message = l10n.t('Skipped creating application insights due to account restrictions or location compatibility.');
         }
         return message;
     }
-    protected getOutputLogSuccess(context: IAppServiceWizardContext): string {
-        const aiName: string = nonNullProp(context, 'newAppInsightsName');
-        let message: string = l10n.t('Successfully created application insights "{0}".', aiName);
+    protected getOutputLogFail = (context: IAppServiceWizardContext) => l10n.t('Failed to create application insights "{0}"', nonNullProp(context, 'newAppInsightsName'));
+    protected getTreeItemLabel(context: IAppServiceWizardContext): string {
+        let message: string = l10n.t('Create application insights "{0}"', nonNullProp(context, 'newAppInsightsName'));
         if (this._skippedCreate) {
```

Comment:
Although not related to your change, for the `if (appInsightsLocation) {}` section below, I think we should handle the condition when there isn't an app insights location.  Since nothing gets created in that case, I think we will want to set the `_skippedCreate` to true so that the skip tree item label is used, otherwise the tree item label will not match what happened.